index.py

```diff
@@ -1,58 +1,183 @@
-from __future__ import print_function
-
 import time
 from typing import Any, Dict, Optional
-
-from fastapi import FastAPI
-
+from fastapi import FastAPI, Depends, HTTPException, status
 from typing import Optional
-
 from sqlmodel import Field, Session, SQLModel, create_engine, select
+from fastapi.security import OAuth2PasswordBearer, OAuth2PasswordRequestForm
+from jose import JWTError, jwt
+from passlib.context import CryptContext
+from datetime import datetime, timedelta
+
+# to get a string like this run:
+# openssl rand -hex 32
+SECRET_KEY = "5ed045a527d2aee3d800e3108463f35300823c322a595d44564c359f6a774037"
+ALGORITHM = "HS256"
+ACCESS_TOKEN_EXPIRE_MINUTES = 30
+
+fake_users_db = {
+    "johndoe": {
+        "id": 31082021184078,
+        "name": "johndoe",
+        "hashed_secret": "$2b$12$EixZaYVK1fsbw1ZfbX3OXePaWxn96p36WQoeG6Lruj3vjPGga31lW",
+        "age": "23",
+        "disabled": False,
+    }
+}
+
+class Token(SQLModel):
+    access_token: str
+    token_type: str
+
+
+class TokenData(SQLModel):
+    username: Optional[str] = None
 
 class User(SQLModel, table=True):
     # TODO change the default value to the timestamp that the user is created
-    id: Optional[int] = Field(default=None, primary_key=True)
+    id: Optional[int] = Field(default=int(time.strftime("%d%m%y%H%M%S", time.localtime())), primary_key=True)
     name: str
     secret_name: str
     age: Optional[str] = None
-      
-engine = create_engine("sqlite:///database.db")
+
+
+connect_args = {"check_same_thread": False}      
```

Comment:
this is quite a dangerous config change to make when the database is on the file system